ansible/configs/kni-osp/pre_infra.yml

```diff
@@ -0,0 +1,33 @@
+- name: Step 000 Pre Infrastructure
+  hosts: localhost
+  gather_facts: false
+  become: false
+  tasks:
+    - debug:
+        msg: "Step 000 Pre Infrastructure"
+
+- name: Step 000 - Create env key
+  hosts: localhost
+  connection: local
+  gather_facts: false
+  become: false
+  tags:
+  - step003
+  - generate_env_keys
+  tasks:
+  - name: Generate SSH keys
```

Comment:
You can avoid these 3 tasks and just use the role.

https://github.com/redhat-cop/agnosticd/blob/development/ansible/roles-infra/infra-local-create-ssh_key/tasks/main.yml